docs/developer-docs/smart-contracts/maintain/resource-limits.mdx

```diff
@@ -46,6 +46,8 @@ The limits depend on the message type as shown in the following table.
 | Wasm custom sections, per canister                                                   | 1MiB        |
 | Wasm custom sections, sections per canister                                          | 16          |
 | Wasm code section, per canister                                                      | 10MiB       |
+| Wasm stable memory, data access per message                                          | 8GiB        |
+| Wasm stable memory, data written per message                                         | 8GiB        |
```

Comment:
I believe this will change soon if this PR is merged, FYI https://gitlab.com/dfinity-lab/public/ic/-/merge_requests/19014